clustering/marathon.py

```diff
@@ -0,0 +1,555 @@
+#!/usr/bin/python
+# (c) 2015, Ludovic Claude <ludovic.claude@laposte.net>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = """
+module: marathon_app
+version_added: "2.1"
+short_description: start and stop applications with Marathon
+description:
+  - Start and stop applications with Marathon.
+author: "Ludovic Claude (@ludovicc)"
+
+options:
+  uri:
+    required: true
+    description:
+      - Base URI for the Marathon instance
+
+  state:
+    required: true
+    choices: [ present, absent, restart, kill ]
+    default: "present"
+    description:
+      - The operation to perform.
+
+  username:
+    required: false
```

Comment:
if `required: false` is defined you must define a default e.g. `default: null` also see http://docs.ansible.com/ansible/developing_modules.html#module-checklist